policyengine_canada/parameters/gov/provinces/ab/tax/income/credits/disability/eligible_age.yaml

```diff
@@ -0,0 +1,10 @@
+description: Alberta provides an additional disability credit to filers below this age
+values:
+  2022-01-01: 18
+metadata:
+  unit: year
+  period: year
+  label: Alberta disability credit age eligibility
+  reference:
+  - title: Alberta Disablitiy credit - Page 2
```

Comment:
try to find this program in the Alberta income tax act - and leave a comment if the program or parts of it are not represented there